File: _sq.py
import sqlite3, os
db = sqlite3.connect("prisma/dev.db")
cur = db.cursor()
cur.execute("SELECT name FROM sqlite_master WHERE type=\"table\"")
tables = [r[0] for r in cur.fetchall()]
print("TABLES:", ", ".join(tables))
for t in tables:
    if t.startswith("Hub"):
        try:
            cur.execute(f'SELECT COUNT(*) FROM "{t}"')
            print(t, "->", cur.fetchone()[0], "rows")
        except Exception as e:
            print(t, "ERR", e)
db.close()

File: _sq.js
const Database = require("better-sqlite3");
try {
  const db = new Database("prisma/dev.db", { readonly: true });
  const tables = db.prepare("SELECT name FROM sqlite_master WHERE type=\"table\"").all();
  console.log("TABLES:", tables.map(t=>t.name).join(", "));
  for (const t of tables) {
    if (t.name.startsWith("Hub")) {
      const c = db.prepare(`SELECT COUNT(*) n FROM "${t.name}"`).get();
      console.log(t.name, "->", c.n, "rows");
    }
  }
} catch(e){ console.error("ERR", e.message); }
